docs/eventing/experimental-features.md

```diff
@@ -51,4 +51,30 @@ data:
 
 ## Features list
 
-<!-- TODO there are no experimental features at the moment -->
+### `KReference.Group` field
+
+**Flag name**: `kreference-group`
+
+**State**: Alpha, disabled by default
+
+When using the `KReference` type to refer to another Knative resource, you can just specify the API `Group` of the resource, instead of the full `APIVersion`.
```

Comment:
```suggestion
When using the `KReference` type to refer to another Knative resource, you can just specify the API `group` of the resource, instead of the full `APIVersion`.
```